### PR TITLE
feat: add instant meeting expiry input

### LINKED ISSUE #15499: [CAL-3951] Add input to change instant meeting duration
### Is your proposal related to a problem?

An input box in org event type settings to add waiting time for instant meeting

Requested by [https://calcominc.slack.com/archives/C04CWEU35NV/p1718298091875219](https://calcominc.slack.com/archives/C04CWEU35NV/p1718298091875219)

<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-3951](https://linear.app/calcom/issue/CAL-3951/add-input-to-change-instant-meeting-duration)</sub>

### PR DESCRIPTION
**Pull request by Udit-takkar**
## What does this PR do?

- Fixes https://github.com/calcom/cal.com/issues/15499
- Fixes CAL-3951 (Linear issue number - should be visible at the bottom of the GitHub issue description)

Screenshot 2024-06-25 at 1 59 54 AM

## Mandatory Tasks (DO NOT REMOVE)

- [x] I have self-reviewed the code (A decent size PR without self-review might be rejected)
- [x] N/A I have added a Docs issue [here](https://github.com/calcom/docs/issues/new) if this PR makes changes that would require a [documentation change](https://docs.cal.com)
- [x] N/A I have added or modified automated tests that prove my fix is effective or that my feature works (PRs might be rejected if logical changes are not properly tested)

## How should this be tested?

1) Go to an org team event type settings
2) Select Instant bookings
3) Turn on Instant Booking toggle 
4) Update the expiry time offset

### CODE REVIEW

**Review by PeerRich — DISMISSED** [review]

File: apps/web/public/static/locales/en/common.json [hunk 1]
```diff
@@ -1043,6 +1043,7 @@
   "seats_nearly_full": "Seats almost full",
   "seats_half_full": "Seats filling fast",
   "number_of_seats": "Number of seats per booking",
+  "set_expiry_time_offset": "Set instant meeting expiry time offset",
```
[the remark this answers is not in the document]
Comment (PeerRich) [reply apps/web/public/static/locales/en/common.json:1046]: or similar

[the remark this answers is not in the document]
Comment (Udit-takkar) [reply apps/web/public/static/locales/en/common.json]: changed to "Set meeting join window (seconds): The time frame in seconds within which host can join and start the meeting. After this period, the meeting join url will expire."

**Review by PeerRich — DISMISSED** [review]

**Comment by PeerRich** [comment]
looks good

easy addition

File: packages/prisma/schema.prisma [hunk 2]
```diff
@@ -133,6 +133,7 @@ model EventType {
   /// @zod.custom(imports.intervalLimitsType)
   durationLimits                       Json?
   isInstantEvent                       Boolean                   @default(false)
+  instantMeetingExpiryTimeOffset       Int                       @default(90)
```
Comment (zomars) [inline packages/prisma/schema.prisma:136]: Nitpick here. Are these seconds? minutes? Can we specify in the column name? I know it's already quite long enough. `instantMeetingExpiryTimeOffsetInSeconds`

Comment (Udit-takkar) [reply packages/prisma/schema.prisma]: done

**Review by zomars — APPROVED** [review]

**Pull request merged**